app/assets/stylesheets/govuk_publishing_components/components/_radio.scss

```diff
@@ -1,5 +1,6 @@
 @import "govuk/components/radios/radios";
 
-.gem-c-radio__heading-text {
+.gem-c-radio__heading-text,
+.govuk-fieldset__legend h1 {
```

Comment:
I'm uneasy whenever I see styles applied to tags. If we really need this gem override on the `govuk-fieldset__heading` can we instead make sure `gem-c-radio__heading-text` is added to the heading (regardless of the level)?

Reply:
Looking at [the original implementation](https://design-system.service.gov.uk/components/radios/) we don't seem to need this custom margin.

Reply:
To my eyes, I would agree that the extra margin isn't needed there @alex-ju but it seems like it was added as requested by one of our designers  https://github.com/alphagov/govuk_publishing_components/pull/1637 
I think it was added to improve readability.

I'm going to think about a better way of doing it, you're right, probably shouldn't apply the styles to the tag.